File: q8.py
topVar = 5


def level1():
    oneVar = 6

    print(topVar + oneVar)

    def level2():

        twoVar = 9
        print(topVar+oneVar+twoVar)

        def level3():
            threeVar = 1
            print(topVar+oneVar+twoVar+threeVar)

        level3()
    level2()


level1()

File: q8.js
var num = 2;

function level1(){

    var num1 = 6;

    console.log(num + num1);
    function level2(){
 
        var num2 = 9;
        console.log(num + num1 + num2);
        function level3(){

            var num3 = 7;
            console.log(num + num1 + num2 + num3);
        }

        level3();
    }

    level2();
}
level1();